test/fixtures/instances/PatientConstraintsEntry.json

```diff
@@ -19,5 +19,6 @@
     "shr.base.EntryType": { "Value" : "http://standardhealthrecord.org/spec/shr/fhir/ActiveFlag" },
     "Value": true
   },
-  "shr.base.EntryType": { "Value": "http://standardhealthrecord.org/spec/shr/fhir/PatientSingleConstraintEntry" }
+  "shr.base.EntryType": { "Value": "http://standardhealthrecord.org/spec/shr/fhir/PatientConstraintsEntry" },
+  "Value": "2017-12-05T23:45:01Z"
```

Comment:
This probably should be truncated to `"2017-12-05"`, since the datatype for `Value` is `date`, not `dateTime` (which is consistent with the FHIR `Patient.birthDate` datatype).